quality-assurance/http-api-mocking-tools.md

```diff
@@ -0,0 +1,52 @@
+# Mocking HTTP Endpoints
+
+This section covers tools in use for creating stub/mock HTTP endpoints/APIs such as for testing a deployed system or testing a service in development in the absence of another required real service.
+
+Notable tools which you can use as an alternative to building your owm mock http APIs are:
+
+  - [Wiremock](http://wiremock.org/)
+  - [Prism](https://stoplight.io/open-source/prism) (Not yet investigated)
+
+## Wiremock
+
+### Simple Mocks
+
+Use wiremock to simplify creating mocks/stubs _unless_ your requirements:
```

Comment:
Can you make this sentence a bit more directive?  "You __should__ use Wiremock for creating stubs etc."

Reply:
Thanks - Have made it more direct from the outset